source/applications/basic/visualization/read_zdf_vis_3d.py

```diff
@@ -114,9 +96,7 @@ def _main():
 
     _display_depthmap(xyz)
 
-    _display_pointcloud(rgba[:, :, 0:3], xyz)
-
-    input("Press Enter to close...")
```

Comment:
Why do we remove this line in this sample but not other samples?

Reply:
Line 119

Reply:
Thought it was not necessary, but I understand why we have it there now! I put it back in